Tests/TestingApps/PurchaseTesterSwiftUI/Shared/ReceiptInspector.swift

```diff
@@ -13,83 +14,166 @@ import ReceiptParser
 struct ReceiptInspectorView: View {
 
     @State private var encodedReceipt: String = ""
-    @State private var parsedReceipt: String = ""
```

Comment:
I'd recommend comparing this file side-by-side, otherwise SwiftUI diff gets hard to read